leakcanary-android/src/main/res/values-de/leak_canary_strings.xml

```diff
@@ -47,6 +47,9 @@
     <string name="leak_canary_permission_not_granted">Bitte gewähre die Storage Berechtigung, andernfalls werden Memory Leaks nicht erkannt.</string>
     <string name="leak_canary_permission_notification_title">Leak erkannt, benötige Berechtigung</string>
     <string name="leak_canary_permission_notification_text">Hier klicken, um Storage Berechtigung für %s zu aktivieren.</string>
+    <string name="leak_canary_stackoverflow_share">Share Leak On Stack Overflow</string>
+    <string name="leak_canary_leak_copied">Leak info copied to clipboard</string>
+    <string name="leak_canary_leak_clipdata_label">LeakCanary Leak Trace</string>
```

Comment:
I do not know German, so I just left this in English for now, as I saw there are a few other strings here not translated yet as well.

Reply:
Yep that's what I've been doing. There has been german contributions for a while now but it's also a pain to maintain.